artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml

```diff
@@ -36,12 +37,12 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.10.1
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     categories: Integration & Delivery
-    containerImage: ""
+    containerImage: projects.registry.vmware.com/vsphere_kubernetes_driver_operator/vdo@sha256:4fb2c8add979f8d06a0aead4286d70f7edf33ec312638a30a68d4fc896e76463
```

Comment:
will this sha change to the latest image that gets pushed to harbor repo

Reply:
No, the reason RH Certification tries to use SHA is that tag's can be deleted or created again, but SHA will be different if the tags has deleted and created again.

so to maintain the consistency, they use SHA